tower-balance/src/pool.rs

```diff
@@ -95,6 +100,7 @@ where
             Level::Normal => Ok(Async::NotReady),
             Level::Low if self.services == 1 => Ok(Async::NotReady),
             Level::Low => {
+                tracing::trace!("removing service for over-provisioned pool");
```

Comment:
...and here?
```suggestion
                tracing::trace!(pool.services = self.services - 1, message = "removing service for over-provisioned pool");
```

(instead of `self.services - 1` we could also just move this down to after we actually remove the service)